README.md

```diff
@@ -2,8 +2,16 @@
 
 **This is not the actual repository for the GitHub website.**
 
-I'm not affiliated with GitHub in any way, except that I use it all
-day long, and almost all my code is hosted there.
+<del>I'm not affiliated with GitHub in any way, except that I use it all
+day long, and almost all my code is hosted there.</del>
+
+<ins>UPDATE: as of April 2020, with [the acquisition of npm, Inc. by
+GitHub](https://blog.npmjs.org/post/612764866888007680/next-phase-montage),
+I am a GitHub employee.  I remain focused on the
+[npm/cli](https://github.com/npm/cli) project, and any opinions expressed
+here belong to the person expressing them at the time they were expressed,
+and should not be taken as official GitHub policy in any way.  I still use
+GitHub all day long, and almost all my code is still hosted there.</ins>
```

Comment:
> and _almost_ all my code is still hosted there

We still haven't reached 100%?  😸

Reply:
@clarkbw He will reach 100% when GitHub becomes Microsoft Git Hub :clap:

Reply:
> @clarkbw He will reach 100% when GitHub becomes Microsoft Git Hub 👏

Microsoft GitHub Professional Edition 365 😉